Makefile

```diff
@@ -1,24 +1,35 @@
-.PHONY: build test
-
 VERSION=$(shell git describe --tags --dirty --always)
 
+.PHONY: build
 build:
 	go build -ldflags "-X 'github.com/conduitio-labs/conduit-connector-stripe.version=${VERSION}'" -o conduit-connector-stripe cmd/connector/main.go
 
+.PHONY: test
 test:
 	go test $(GOTEST_FLAGS) -race ./...
 
+.PHONY: generate
+generate:
+	go generate ./...
+
+.PHONY: lint
 lint:
-	golangci-lint run -c .golangci.yml --go=1.18
+	golangci-lint run -v
 
+.PHONY: dep
 dep:
 	go mod download
 	go mod tidy
 
+.PHONY: mockgen
 mockgen:
 	mockgen -package mock -source source/source.go -destination source/mock/source.go
 	mockgen -package mock -source source/iterator/iterator.go -destination source/iterator/mock/iterator.go
```

Comment:
I'd like to suggest replacing this with `go:generate` directives  that are put into source files. That way, we can have a single command, `make generate`, to generate all the mocks.